lib/gitter-adapter.js

```diff
@@ -215,6 +217,12 @@ Adapter.prototype.subscribeToRoom = function(room) {
 
 };
 
+Adapter.prototype.queueMessage = function(target, message) {
+    this.sendPromiseChain = this.sendPromiseChain.then(function() {
+        return this.sendMessage(target, message);
+    }.bind(this));
```

Comment:
Just noting: Consider a rejected promise breaking the chain for everything.

I think we are all good though because of the `.catch` at the end of `sendMessage` already.